pkg/subctl/cmd/show_connections.go

```diff
@@ -36,30 +36,35 @@ func getConnectionsStatus(config *rest.Config) []connectionStatus {
 	submarinerClient, err := submarinerclientset.NewForConfig(config)
 	exitOnError("Unable to get the Submariner client", err)
 
+	SubmarinerInstalled = true
 	var status []connectionStatus
 
 	existingCfg, err := submarinerClient.SubmarinerV1alpha1().Submariners(OperatorNamespace).Get(submarinercr.SubmarinerName, v1.GetOptions{})
 	if err != nil {
-		exitOnError("error reading from submariner client", err)
+		if ! SubmarinerNotInstalled(err) {
```

Comment:
This "double negative" is a knows anti patten, why not name the function "submarinerMissing" and the "if" here will only be true if it is.

Reply:
The logic here is ```if there is error and submariner is installed, exitOnError otherwise exit current cluster and continue with other clusters``` Function name, IMO, won't matter. Instead, I changed the function to ```SubmarinerIsInstalled```.